presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchConfig.java

```diff
@@ -48,11 +48,14 @@
 {
     public enum Security
     {
-        AWS
+        AWS,
+        X_PACK
     }
 
     private String host;
     private int port = 9200;
+    private String username;
```

Comment:
Let's put these in a separate config similar to how we handle `io.prestosql.elasticsearch.AwsSecurityConfig`